src/components/subscriptions/SubscriptionExpirationBanner.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from 'react';
+import { MailtoLink } from '@edx/paragon';
+
+import StatusAlert from '../../components/StatusAlert';
+import { SubscriptionContext } from './SubscriptionData';
+import {
+  SUBSCRIPTION_DAYS_REMAINING_MODERATE,
+  SUBSCRIPTION_DAYS_REMAINING_SEVERE,
+  SUBSCRIPTION_DAYS_REMAINING_EXCEPTIONAL,
+} from './constants';
+
+export default function SubscriptionExpirationBanner() {
+  const { details } = useContext(SubscriptionContext);
+  const { daysUntilExpiration } = details;
+
+  const renderMessage = () => (
+    <React.Fragment>
+      Your subscription is {daysUntilExpiration} days from expiration.
+      Contact the edX Customer Success team at
+      {' '}
+      <MailtoLink to="customersuccess@edx.org">customersuccess@edx.org</MailtoLink>
+      {' '}
+      to extend your contract.
+    </React.Fragment>
+  );
+
+  if (daysUntilExpiration > SUBSCRIPTION_DAYS_REMAINING_MODERATE) {
+    return null;
+  }
+
+  let dismissible = true;
+  let alertType = 'info';
+  if (daysUntilExpiration <= SUBSCRIPTION_DAYS_REMAINING_SEVERE) {
+    alertType = 'warning';
+  }
+  if (daysUntilExpiration <= SUBSCRIPTION_DAYS_REMAINING_EXCEPTIONAL) {
+    dismissible = false;
+    alertType = 'danger';
```

Comment:
Are these alertTypes not in a constants file anywhere? I would move them into a constants file and then import them if they aren't already existing in a constants file somewhere.

Reply:
These are just the bootstrap "themes" https://getbootstrap.com/docs/4.0/utilities/colors/so I don't think it's especially worthwhile to use constants here whereas our other frontends do not